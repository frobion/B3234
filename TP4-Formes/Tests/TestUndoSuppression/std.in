R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
DELETE Segment1
UNDO
LIST
PC PolygonConvex1 1 1 3 -5 -2 -1
CLEAR
UNDO
LIST
EXIT
