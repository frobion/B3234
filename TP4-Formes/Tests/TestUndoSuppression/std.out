OK
OK
OK
OK
R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
OK
OK
OK
R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
PC PolygonConvex1 1 1 3 -5 -2 -1

